src/components/organisms/UiTabs/UiTabs.stories.ts

```diff
@@ -0,0 +1,337 @@
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { useArgTypes } from '@sb/helpers';
+import { content as contentArgTypes } from '@sb/helpers/argTypes';
+import { withModelValue } from '@sb/decorators';
+import type { TabsProps } from '@index';
+import type { TabsItemAttrsProps } from './_internal/UiTabsItem.vue';
+import UiTabs from '@/components/organisms/UiTabs/UiTabs.vue';
+import UiTabsItem from '@/components/organisms/UiTabs/_internal/UiTabsItem.vue';
+import UiText from '@/components/atoms/UiText/UiText.vue';
+
+type TabsArgsType = TabsProps & {
+  items?: TabsProps[],
+  content?: Record<string, string>,
+  modifiers?: string[]
+}
+
+type TabsMetaType = Meta<TabsArgsType>;
+type TabsStoryType = StoryObj<TabsArgsType>
+
+const complexItemsData: TabsItemAttrsProps[] = [
+  {
+    name: 'search',
+    title: 'Search',
+    'data-testid': 'search',
+    buttonTabAttrs: { 'data-testid': 'search-button' },
+    contentAttrs: { 'data-testid': 'search-content' },
+  },
+  {
+    name: 'point',
+    title: 'Point on the body',
+    buttonTabAttrs: { 'data-testid': 'point-button' },
+    contentAttrs: { 'data-testid': 'point-content' },
+  },
+  {
+    name: 'drop',
+    title: 'Drop body point',
+    buttonTabAttrs: { 'data-testid': 'drop-button' },
+    contentAttrs: { 'data-testid': 'drop-content' },
+  },
+];
+
+const basicTemplate = `<UiTabs
```

Comment:
related to `basicTemplate`, `basicScript` and others
for me is hard to readed. When I go to the line `export const Basic..` I want to find a full setup there. Or after export where I have `Basic.parameters` for example. 

code should be I `Basic.docs.source.code`